private/testing-library/index.ts

```diff
@@ -24,7 +25,7 @@ type SkuCommand =
 
 // Default configuration for cli-testing-library
 const DEFAULT_CONFIG: Partial<Config> = {
-  asyncUtilTimeout: 5000,
+  asyncUtilTimeout: 10000,
```

Comment:
5000ms wasn't quite cutting it. I was often getting test failures when running all `sku-test` tests, but no failures when running tests individually, so I bumped up this timeout.